task_1_1_1

grubrescue opened the pull request.

grubrescue committed:
heapsort partially done

grubrescue committed:
harder better faster stronger

grubrescue committed:
changed folder structure

grubrescue committed:
little build.gradle change for the workkflow

grubrescue committed:
stupid mistake in .gitignore :(

grubrescue committed:
now fixed yay

grubrescue committed:
followed some reviewbot's recomendations

grubrescue committed:
restructured it again, improved commentary

grubrescue committed:
little misfortune

grubrescue committed:
renamed smth

grubrescue committed:
here we go again...

grubrescue committed:
phantom commit to rerun workflow

grubrescue committed:
updated workflow

grubrescue committed:
exception handling in a sort() method and some cosmetic changes

molo4evan commented on task_1_1_1/build.gradle:
```diff
@@ -0,0 +1,24 @@
+plugins {
```
> Кроме настроенной системы сборки gradle необходимо предоставить преподавателю скрипт без использования системы сборки для: компиляции, генерации документации и запуска решения

Нужен bat либо sh, в зависимости от ключей запускающий ту или иную команду
Для компиляции см. `javac`, для генерации доков - `javadoc`, для запуска программы - `java`

В остальном - все прекрасно

grubrescue committed:
a looot of changes
+now we have main
+now we have shell build script
+now we have readme for shell build script
+and some minor changes in heap
-and no tests for main (do we need them?)

grubrescue committed:
Update README.md

molo4evan commented:
Зарезолви конфликты, в остальном претензий нет

grubrescue committed:
Merge branch 'master' into task-1-1-1
